include/oneapi/dnnl/dnnl.hpp

```diff
@@ -4063,21 +4063,51 @@ struct primitive_attr : public handle<dnnl_primitive_attr_t> {
     ///
     /// @param mask_desc Output memory descriptor of a dropout mask.
     void get_dropout(memory::desc &mask_desc) const {
+        memory::data_type seed_dt;
+        bool use_offset, use_host_scalars;
+        get_dropout(mask_desc, seed_dt, use_offset, use_host_scalars);
+        // TODO: add checks that seed_dt = f32, use_offset=use_host_scalars=false;
+        //       or call C API with nullptr to trigger checks internally?
+    }
```

Comment:
I don't recall we update previously defined C++ API calls. I'd keep v1 dropout call for validation purpose. Same for setter.